bot/exts/moderation/silence.py

```diff
@@ -47,7 +48,16 @@ class SilenceNotifier(tasks.Loop):
     """Loop notifier for posting notices to `alert_channel` containing added channels."""
 
     def __init__(self, alert_channel: TextChannel):
-        super().__init__(self._notifier, seconds=1, minutes=0, hours=0, count=None, reconnect=True, loop=None)
+        super().__init__(
+            self._notifier,
+            seconds=1,
+            minutes=0,
+            hours=0,
+            count=None,
+            reconnect=True,
+            loop=None,
+            time=MISSING
```

Comment:
Huh? Shouldn't `MISSING` be the keyword argument's default already?

Reply:
> Huh?
That was also my reaction :D but apparently due to how the API changed you have to set it explicitly